resources/js/components/ResourceCard.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <div class="card">
+        <img class="card-img-top"
+              :src="resource.image_url || '/img/placeholder.png'"
+              alt="Card image cap"/>
+        <div class="card-body">
+            <p class="font-weight-bold mb-1">{{resource.title}}</p>
+            <span class="text-muted">
+                {{resource.description}}
+            </span>
+        </div>
+        <div class="card-footer pt-0 d-flex px-2 pb-2 align-items-center">
+            <a :href="resource.link" class="btn btn-link">Learn more</a>
+            <div class="ml-auto">
+                <button type="button"
+                        class="btn btn-link-dark btn-sm"
+                        v-tooltip="'Delete'"
+                        :disabled="deleting"
+                        @click.prevent="destroy()"
+                >
+                    <inline-spinner v-if="deleting"/>
+                    <icon name="trash" v-if="!deleting"/>
+                </button>
+                <button type="button"
+                        class="btn btn-link-dark btn-sm"
+                        v-tooltip="'Edit'"
+                        @click.prevent="update()">
+                    <icon name="create" class="icon-normal"/>
+                </button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import Icon from '../components/Icon'
+import InlineSpinner from '../components/InlineSpinner'
+
+export default {
+  name: 'ResourceCard',
+
+  components: {
+    Icon,
+    InlineSpinner,
+  },
+
+  props: {
+    resource: {required: true, default: null, type: Object},
+  },
+
+  data() {
+    return {
+      deleting: false,
+    }
+  },
+
+  methods: {
+    update() {
+      this.$emit('update')
+    },
+
+    destroy() {
+      this.$confirm({
+        title    : `Delete ${this.resource.title}?`,
+        text     : `Deleting the resource is a permanent action.`,
+        onConfirm: async () => {
+          this.deleting = true
+          try {
+            await axios.delete(`/web/resources/${this.resource.id}`)
+            this.$emit('delete')
+          } catch (e) {
+            this.$alert(`Unable to delete ${this.resource.title}. Please try refreshing the page.`)
+            console.error(e)
+          }
+        },
```

Comment:
You need to make sure the deleting variable is reset after we are done 
```suggestion
          this.deleting = false
        },
```